src/components/Services.spec.jsx

```diff
@@ -14,6 +14,7 @@ jest.mock('cozy-client', () => ({
   ...jest.requireActual('cozy-client'),
   useAppsInMaintenance: jest.fn()
 }))
+// eslint-disable-next-line react/display-name
```

Comment:
let's add in eslintrc.json
```
overrides: [
{       files: ['**.spec.jsx'],       rules: { 'react/display-name: 'off' },     },
]
```

Reply:
I would disagree here, I think we should keep untouched eslintrc files in our apps. I commented the line because I didn't want to modify it.

The real fix IMO is to refactor the code, which I shall do when I can